sakura_core/CProfile.cpp

```diff
@@ -250,10 +250,10 @@ bool CProfile::WriteProfile(
 		vecLine.push_back( LTEXT(";") + wstring( pszComment ) );		// //->;	2008/5/24 Uchi
 		vecLine.push_back( LTEXT("") );
 	}
-	for(auto iter = m_ProfileData.begin(); iter != m_ProfileData.end(); iter++ ) {
+	for(auto iter = m_ProfileData.cbegin(); iter != m_ProfileData.cend(); iter++ ) {
```

Comment:
```suggestion
	for(const auto iter = m_ProfileData.cbegin(); iter != m_ProfileData.cend(); iter++ ) {
```
付けなくても影響ないですが見やすくなる気がします。

Reply:
increment してるので const つけるとコンパイルエラーになります。

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2678	binary '++': no operator found which takes a left-hand operand of type 'const std::_Vector_const_iterator<std::_Vector_val<std::_Simple_types<_Ty>>>' (or there is no acceptable conversion)	sakura	D:\projects\sakura\sakura_core\CProfile.cpp	253	
```